src/routes/(app)/namespaces/[namespace]/workflows/[workflow]/[run]/user-metadata/+page.svelte

```diff
@@ -0,0 +1,49 @@
+<script>
+  import MetadataEvents from '$lib/components/workflow/metadata/metadata-events.svelte';
+  import WorkflowCurrentDetails from '$lib/components/workflow/metadata/workflow-current-details.svelte';
+  import WorkflowSummaryAndDetails from '$lib/components/workflow/metadata/workflow-summary-and-details.svelte';
+  import { translate } from '$lib/i18n/translate';
+  import { groupEvents } from '$lib/models/event-groups';
+  import { fullEventHistory } from '$lib/stores/events';
+  import { workflowRun } from '$lib/stores/workflow-run';
+
+  const { workflow } = $derived($workflowRun);
+
+  const relevantEvents = $derived(
+    $fullEventHistory.filter((event) => {
+      if (event.userMetadata?.summary || event.userMetadata?.details) {
+        return true;
+      }
+
+      return false;
+    }),
+  );
+
+  const eventGroups = $derived(
+    groupEvents(
+      relevantEvents,
+      'ascending',
+      workflow?.pendingActivities,
+      workflow?.pendingNexusOperations,
+    ),
+  );
+</script>
+
```

Comment:
Same here. Let's put all this logic in <WorkflowUserMetadata /> page and add PageTitle here


```
<PageTitle
  title={`${translate('workflows.user-metadata-tab')} | ${page.params.workflow}`}
  url={page.url.href}
/>
<WorkflowPaddedLayout>
 <WorkflowUserMetadata />
</WorkflowPaddedLayout>

```

Reply:
I made the WorkflowUserMetadata into a component, but opted to not use the WorkflowPaddedLayout because the formatting is a little bit different.